R/prepare-admin-panel.R

```diff
@@ -1,9 +1,7 @@
 # define function to get username
 get_user <- function(session) {
-  username <- shiny::isolate(shiny::parseQueryString(session$clientData$url_search)$username)
-  if (is.null(username)) username <- "unknownUser"
-  shiny::req(username)
-  return(username)
+  if (is.null(session) || is.null(session$user)) return("anonymous")
+  session$user
```

Comment:
Since `get_user()` is no longer used inside `analytics_server()`, I think we can safely remove this. I tested by removing this function. It still seems to be working, but please try to confirm.

Reply:
Agree! I wasn't sure about this one